### PR TITLE
fix GEARPUMP-169, upgrade sbt-assembly and shade

### PR DESCRIPTION
**Pull request by manuzhang**

### CODE REVIEW

**Comment by manuzhang** [comment]
I also removed cloudera dependencies

File: project/Build.scala [hunk 1]
```diff
@@ -104,6 +99,7 @@ object Build extends sbt.Build {
         System.getenv().get("SONATYPE_PASSWORD")),
 
       pomIncludeRepository := { _ => false },
+      test in assembly := {},
```
Comment (huafengw) [inline project/Build.scala]: I think we can move this line to the myAssemblySettings?

**Comment by huafengw** [comment]
travis build fail

**Comment by manuzhang** [comment]
the test failure is not related

```
[0m[[0minfo[0m] [0m[32mWorker[0m[0m
[0m[[0minfo[0m] [0m[31m- should register worker address to master when started. *** FAILED ***[0m[0m
[0m[[0minfo[0m] [0m[31m  java.lang.AssertionError: assertion failed: timeout (25 seconds) during expectMsg while waiting for RegisterNewWorker[0m[0m
[0m[[0minfo[0m] [0m[31m  at scala.Predef$.assert(Predef.scala:170)[0m[0m
[0m[[0minfo[0m] [0m[31m  at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:349)[0m[0m
[0m[[0minfo[0m] [0m[31m  at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:335)[0m[0m
[0m[[0minfo[0m] [0m[31m  at akka.testkit.TestKit.expectMsg(TestKit.scala:734)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.apache.gearpump.cluster.main.MainSpec$$anonfun$1.apply$mcV$sp(MainSpec.scala:73)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.apache.gearpump.cluster.main.MainSpec$$anonfun$1.apply(MainSpec.scala:56)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.apache.gearpump.cluster.main.MainSpec$$anonfun$1.apply(MainSpec.scala:56)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.scalatest.Transformer$$anonfun$apply$1.apply$mcV$sp(Transformer.scala:22)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)[0m[0m
[0m[[0minfo[0m] [0m[31m  at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)[0m[0m
```

**Comment by kkasravi** [comment]
+1

**Pull request closed without merging**